crates/ruff_python_formatter/src/expression/expr_await.rs

```diff
@@ -20,7 +20,7 @@ impl FormatNodeRule<ExprAwait> for FormatExprAwait {
             [
                 text("await"),
                 space(),
-                maybe_parenthesize_expression(value, item, Parenthesize::IfRequired)
+                maybe_parenthesize_expression(value, item, Parenthesize::IfBreaks)
```

Comment:
I dislike that `yield` and `await` expressions are inconsistent. Should we keep it as is and mark this an intentional deviation (I would still merge the change because using `IfBreaks` instead of `IfRequired` cleans up the expression logic)